examples/dynamic/views/main.jsx

```diff
@@ -0,0 +1,10 @@
+var React = require('react');
+var Content = React.createFactory(require('./Content'));
```

Comment:
You should actually see a warning about passing a function to `createElement` when you are using `createFactory` + JSX. Let's drop the `createFactory` call.